app/models/round.rb

```diff
@@ -0,0 +1,56 @@
+class Round < ApplicationRecord
+  before_create :set_position
+  before_destroy :update_positions
+
+  validates :round_type, presence: true
+  validate :unique_round_for_player_in_game
+
+  belongs_to :game_player
+  has_one :game, through: :game_player
+
+  enum round_type: {
+    rentz_minus: 'Rentz -',
+    rentz_plus: 'Rentz +',
+    totale_minus: 'Totale -',
+    totale_plus: 'Totale +',
+    king: 'King of hearts',
+    ten: 'Ten of clubs',
+    queens: 'Queens',
+    diamonds: 'Diamonds'
+  }
+
+  def decrement_position
+    self.position -= 1
+    save!
+  end
```

Comment:
Isn't this provided by the `acts_as_list` gem?

Reply:
If do not overwrite it I get an `NoMethodError` when I try to destroy all rounds at once. When I destroy them one by one it's working without overwriting.